ClipVessel/ClipVessel.py

```diff
@@ -0,0 +1,800 @@
+import os 
+import unittest
+import logging
+import vtk, qt, ctk, slicer
+import numpy as np
+from slicer.ScriptedLoadableModule import *
+from slicer.util import VTKObservationMixin
+import vtkvmtkComputationalGeometryPython as vtkvmtkComputationalGeometry
+
+"""
+  ClipVessel
+"""
+
+class ClipVessel(ScriptedLoadableModule):
+  """Uses ScriptedLoadableModule base class, available at:
+  https://github.com/Slicer/Slicer/blob/master/Base/Python/slicer/ScriptedLoadableModule.py
+  """
+
+  def __init__(self, parent):
+    ScriptedLoadableModule.__init__(self, parent)
+    self.parent.title = "Clip Vessel"
+    self.parent.categories = ["Vascular Modeling Toolkit"]
+    self.parent.dependencies = []
+    self.parent.contributors = ["David Molony (NGHS)", "Andras Lasso (PerkLab)"]
+    self.parent.helpText = """
+This module clips a surface model given a VMTK centerline and markups indicating where the model will be clipped. The first marker indicates the inlet. Optionally, the user can cap and add flow extensions.
+    <a href="https://github.com/vmtk/SlicerExtension-VMTK/">here</a>.
```

Comment:
add url to the module documentation (not just the repository) to make the documentation easier to find